Dockerfile

```diff
@@ -1,8 +1,20 @@
-FROM fabric8/java-jboss-openjdk8-jdk:1.5.1
+FROM --platform=$BUILDPLATFORM eclipse-temurin:18-jdk-focal as builder
+
+WORKDIR /app
+COPY gradle/ gradle/
+COPY gradlew .
+COPY settings.gradle .
+COPY build.gradle .
+COPY src src/
+
+RUN ./gradlew shadowJar --no-daemon
+
+FROM eclipse-temurin:17.0.8.1_1-jre
```

Comment:
why is this using older version than builder?

Reply:
I can use the same version for the builder. For this image, I used the `17.0.8.1_1-jre` tag because it supports `linux/s390x`. Java 8 images don't support it.